src/components/samples/SamplesDetailContent.js

```diff
@@ -209,11 +209,13 @@ const SamplesDetailContent = ({samplesByID, containersByID, individualsByID, use
 };
 
 function renderTimelineLabel(version, usersByID) {
-  const user = usersByID[version.revision.user];
+  const user = usersByID[version.revision.user]
+  const username = user ? user.username : ""  // in case usersByID is still loading asynchronously
+
```

Comment:
`const username = user?.username` would be preferable if you're going to default to an empty string (which I dont recommend).

The comment is not necessary if you render a loading indicator, e.g. `const username = user?.username ?? "Loading..."`